### PR TITLE
[STONE-774] Use redhat-appstudio-appdata-staging on staging

### PR DESCRIPTION
**Pull request by johnmcollier**
Fixes https://issues.redhat.com/browse/STONE-774

Updates the production and staging overlays to specify the correct GitHub org in HAS for GitOps repository. Production will use https://github.com/redhat-appstudio-appdata and staging will use https://github.com/redhat-appstudio-appdata-staging

### CODE REVIEW

**Review by maysunfaisal — APPROVED** [review]

File: components/has/staging/kustomization.yaml [hunk 1]
```diff
@@ -3,3 +3,6 @@ kind: Kustomization
 resources:
   - ../base
   - ../base/external-secrets
+
+patches:
+  - ./gitops-org-patch.yaml
```
Comment (kim-tsao) [inline components/has/staging/kustomization.yaml:8]: Do we need to have the token patch and target name as it's configure in the production kustomization.yaml file?

Comment (johnmcollier) [reply components/has/staging/kustomization.yaml:8]: No, the default external secret resource exists under the base folder, and points to staging.

**Pull request merged**